test/semantics/dosemantics05.f90

```diff
@@ -0,0 +1,31 @@
+! Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+
+! C1127 -- The DEFAULT (NONE) locality-spec shall not appear more than once in a
+! given concurrent-locality.
+PROGRAM dosemantics05
+  IMPLICIT NONE
+  INTEGER :: ivar
+
+! This one works
+  DO CONCURRENT (ivar = 1:10) DEFAULT (NONE)
+    PRINT *, "ivar is: ", ivar
+  END DO
+
+!ERROR: only one DEFAULT(NONE) may appear
+  DO CONCURRENT (ivar = 1:10) DEFAULT (NONE) DEFAULT (NONE)
```

Comment:
This can be a warning.  It's benign.

Reply:
And please document as an extension.

Reply:
I vote that it is better to follow the standard in this case.  Why add an unnecessary laxness for a new feature without a track record of needing it, with the associated hassles (like documentation)?

Reply:
I agree with Val. There are no implementations that I could find that allow this.

Reply:
I also agree with Val.  If I wrote this code, I'd want to be notified that there was something wrong.  On the other hand, if
we're warning for non-standard usage, I would be turning on the option to do that and to treat warnings as errors.

Reply:
I'm not saying that we shouldn't emit a message.  It just doesn't need to be fatal by default.  Why should it be?  Nothing bad is going to happen if compilation proceeds.

Reply:
Actually, this sounds right to me.  If there's no objection, I'll change the code to check to make this an error only in the case where the -Mstandard and -Werror options are in effect.

Reply:
You shouldn't have to change this code to accomplish that result.

Reply:
@klausler , I'm not sure what you mean by not having to change the code.  This pull request only adds a test.  If we don't change the current behavior of the compiler, it will issue an error message when two ```DEFAULT (NONE)``` clauses appear in a DO CONCURRENT header.  If I understand your previous comments correctly, that's what you want to change, and that's what I'm proposing to do.

Reply:
I was responding to your "I'll change the code to check to make this an error only in the case where the -Mstandard and -Werror options are in effect.", which I understood to mean that you're changing the code that emits the message.